### DISCUSSION TITLE
Relationships in the published section

### DISCUSSION THREAD

**Discussion by EronitaB** [Budibase/budibase#2509] [Help] [answered]
Hi there,
I have one question about the app. In the first photo is the app that I as an admin see, as I click the clients section, each client is clickable and sends me to the informations of the client.
Meanwhile in the second photo there is the 'published app' section. But the clients aren't clickable, after clicking them they don't send me to the client informations.
Any idea how to make it work also in the published section?

Thanks,
Eronita

![image](https://user-images.githubusercontent.com/77198959/131525128-e1515b0f-f765-4c86-b2ba-33e465c050ed.png)

![image](https://user-images.githubusercontent.com/77198959/131525224-439e8b08-cf09-429e-8955-e636d15d5c52.png)

**Comment by mjashanks** [comment, marked as answer]
Hi @EronitaB 

Unfortunately, this is not possible at the moment.

I attempted to recreate this, using a new Button in the row, with a `Navigate To` action. It did not work as expected.

![image](https://user-images.githubusercontent.com/3524181/131681330-8c7f82b1-a953-48d2-814b-975dc6cd847c.png)

I think we should add the ability to use the ClientId in bindings, like `/clients/{{ Projects Table.Projects.Client_id }}`. Not possible right now though.

**Reply by EronitaB** [reply]
@mjashanks thanks a lot :)
